### ISSUE TITLE
Does videojs big play button work with wavesurfer plugin?

### ISSUE THREAD

**Issue by ignl** [collab-project/videojs-wavesurfer#120]
Hello,

I setup my player with big play button, but once I add wavesurfer plugin it disappears. I am using latest versions of videojs and wavesurfer. Is it supported, maybe I need to do something additionally for it to work?

**Comment by thijstriemstra** [comment]
@ignl yea it's always been hidden, defined here: https://github.com/collab-project/videojs-wavesurfer/blob/master/src/js/videojs.wavesurfer.js#L73

Setting the `bigPlayButton` option to `true`:

```javascript
var options = {
    controls: true,
    autoplay: false,
    loop: false,
    muted: false,
    fluid: false,
    width: 600,
    height: 300,
    bigPlayButton: true,
   // etc
```

doesn't help because the code I pointed out will always hide it. If you really want to use the button, comment out the code and set `bigPlayButton` to `true`. I tried this but it'll screw up the player flow and give you a unclickable button:

<img width="655" alt="Screenshot 2020-12-17 at 01 38 53" src="https://user-images.githubusercontent.com/305679/102423149-a5c50280-4008-11eb-81b3-b5d8353f356b.png">

So yea, I could modify the code so it doesn't auto-hide and relies on the `bigPlayButton` option but then again, it'll always be `false` since it's not functioning. Thoughts?
